src/Annotations/AbstractAnnotation.php

```diff
@@ -366,10 +366,25 @@ public function jsonSerialize()
                     }
                 }
             }
+            if (property_exists($this, 'nullable') && $this->nullable === true) {
+                if ($this->nullable !== $defaultValues['nullable']) {
```

Comment:
if `nullable` is `true` does it matter what the default value was/is and if it differs?

Reply:
I think the if statement on line 370 can go altogether. I do not particularly like the defaultValues stuff anyway. I'd rather is `Generator::isDefault()` although that is a different matter.

Reply:
i took inspiration of some code above, I don't know if the default value is important or not. But it looks like the code changed since then. If the default value is true I suppose we still need to do the same trick so we could remove the defaultValue check

Reply:
It doesn't really matter what the default is - if it is `true` then its true :).  In practice the default is `Generator::DEFAULT` and `false` will also be different from the default value... so, yeah, this can go IMO.